_includes/v21.2/sidebar-data/develop.json

```diff
@@ -0,0 +1,578 @@
+          "items": [
+            {
+              "title": "Reusable Views",
+              "urls": [
+                "/${VERSION}/views.html"
+              ]
+            },
+            {
+              "title": "Subqueries",
+              "urls": [
+                "/${VERSION}/subqueries.html"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "Create Temporary Tables",
+          "urls": [
+            "/${VERSION}/temporary-tables.html"
+          ]
+        },
+        {
+          "title": "Paginate Results",
+          "urls": [
+            "/${VERSION}/pagination.html"
+          ]
+        },
+        {
+          "title": "Read Historical Data",
+          "items": [
+            {
+              "title": "AS OF SYSTEM TIME",
+              "urls": [
+                "/${VERSION}/as-of-system-time.html"
+              ]
+            },
+            {
+              "title": "Follower Reads",
+              "urls": [
+                "/${VERSION}/follower-reads.html"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "Query Spatial Data",
+          "urls": [
+            "/${VERSION}/query-spatial-data.html"
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Transactions",
+      "urls": [
+        "/${VERSION}/transactions.html"
+      ]
+    },
+    {
+      "title": "Debug Your Application",
+      "items":
+      [
+        {
+          "title": "Log Events",
+          "urls": [
+            "/${VERSION}/logging-overview.html"
+          ]
+        },
+        {
+          "title": "Monitor CockroachDB Apps in the DB Console",
+          "urls": [
+            "/${VERSION}/ui-overview.html"
+          ]
+        },
+        {
+          "title": "Troubleshoot Common Problems",
+          "urls": [
+            "/${VERSION}/error-handling-and-troubleshooting.html"
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Optimize Performance",
+      "items":
+      [
+        {
+          "title": "Overview",
+          "urls": [
+            "/${VERSION}/make-queries-fast.html"
+          ]
+        },
+        {
+          "title": "Performance Best Practices",
+          "urls": [
+            "/${VERSION}/performance-best-practices-overview.html"
+          ]
+        },
+        {
+          "title": "Use the EXPLAIN statement",
+          "urls": [
+            "/${VERSION}/sql-tuning-with-explain.html"
+          ]
+        },
+        {
+          "title": "Performance Tuning Recipes",
+          "urls": [
+            "/${VERSION}/performance-recipes.html"
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Example Apps",
+      "items": [
+        {
+          "title": "Overview",
+          "urls": [
+            "/${VERSION}/example-apps.html"
+          ]
+        },
+        {
+          "title": "Build a Hello World Application",
+          "items": [
+            {
+              "title": "Go",
```

Comment:
I think this is too terse for SEO. Couldn't it be Build an Application in Go. Ditto for the other languages.

Reply:
Leaving for now, as prefixing the topics would appear as:

```
Building a Hello World Application in Go
Building a Hello World Application in Java
Building a Hello World Application in JavaScript (Node.js)
Building a Hello World Application in Python
```

I'm not convinced that adds much value for navigation users, so it would only be useful for SEO. But if we decided differently we can iterate on this in a follow-up PR.